nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/RunWorker.cs

```diff
@@ -305,6 +359,16 @@ string GetFullRepoPath(string path)
         return updatedDependencyList;
     }
 
+    private static string GetPackagesConfigFromCsprojLocation(string fullRepoPath)
+    {
+        int lastSlash;
+        fullRepoPath = Path.GetFullPath(fullRepoPath).NormalizePathToUnix().EnsurePrefix("/");
+
+        lastSlash = fullRepoPath.LastIndexOf('/');
```

Comment:
You can use `Path.GetDirectoryName()` to do this work for you.  Something like `Path.Combine(Path.GetDirectoryName(fullRepoPath), "packages.config")`